Add hints for using Syncthing with Obsidian (#243)
obsidian-community/obsidian-hub: 02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/Syncthing.md

# Syncthing

Official website: https://syncthing.net/
Documentation: https://docs.syncthing.net/
Cost: free, open source
Available for: [[Windows Tools|Windows]], [[MacOS Tools|MacOS]], [[Linux Tools|Linux]], [[Android Apps|Android]]

A peer-to-peer file synchronisation application, which can be used to keep your Obsidian vault synchronised across multiple devices almost instantaneously.

## Configuration for Obsidian
You'll probably want the following files to be excluded from synchronisation in your `.stignore`. If you're not using the [[metadata-extractor|Metadata Extractor]] plugin, you'll only need to exclude `workspace`.
```
// most important one. this keeps track of your open panes and files in the app
.obsidian/workspace
// Metadata Extractor generates these automatically, so you shouldn't sync them
.obsidian/plugins/metadata-extractor/allExceptMd.json
.obsidian/plugins/metadata-extractor/metadata.json
.obsidian/plugins/metadata-extractor/tags.json
```
You *can* exclude your entire `.obsidian` directory instead for simplicity, but in that case your plugins and configuration will not be synchronised at all.

> Note that Syncthing does *not* synchronise the `.stignore` itself, so you will need to set up your ignore patterns on each device separately. We recommend saving a copy of your `.stignore` elsewhere in your Obsidian vault, so it can simply be copied to set up a new device smoothly.

Reviewer: > We recommend saving a copy of your `.stignore` elsewhere in your Obsidian vault, so it can simply be copied to set up a new device smoothly.

Not sure whether I'm too picky, but maybe `we` shouldn't be used in that case?

```suggestion
Note that Syncthing does *not* synchronise the `.stignore` itself, so you will need to set up your ignore patterns on each device separately. If you save a copy of your `.stignore` elsewhere in your Obsidian vault, it can simply be copied to set up a new device smoothly.
```
